dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -750,6 +749,31 @@ def school_info_attr
       end
     end
 
+    # @override
+    def find_default_workshop
+      get_first_selected_workshop || super
+    end
+
+    def get_first_selected_workshop
+      hash = sanitize_form_data_hash
+      workshop_ids = hash[:regional_partner_workshop_ids]
+      return nil unless workshop_ids && !workshop_ids.empty?
```

Comment:
Think you can just do `workshop_ids.try(&:empty?)`

Reply:
That won't work because I want to return if it's nil or empty (not the `!`), but this will: `workshop_ids.try(:any?)` and is definitely more readable 👍